dcmorales/digital-keyboard-tunes
# Feature: add custom options

## Description
Created a header that contains keyboard-settings. The settings can update the key, octave, and waveform. Depending on the provided values, the keyboard-selected will update to display the correct keys. The updates are done through the context provider. 

## Steps to test / reproduce
From the header component, toggle the settings on. Update the key or octave values and notice the keys update on the keyboard-selected.

## Types of changes



- [ ] Task (non-breaking change that is not a bug or feature)

## Reminders



- [x] I have tested that my code builds successfully
- [x] I have pulled the latest version of this branch's base branch (i.e. `main`)
- [x] I have updated the README or inline docs if needed
- [x] I have added a short, descriptive title to this PR

## Other comments
